datafusion/optimizer/src/test/user_defined.rs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This appears to be duplicating the file from core, instead of moving it. Is this intentional?